app/javascript/controllers/count_controller.js

```diff
@@ -0,0 +1,25 @@
+import { Controller } from "@hotwired/stimulus"
+
+export default class extends Controller {
+  static targets = ["counter", "display"]
+  static values = {
+    selected: { type: Number, default: 0 },
+  }
+
+  connect() {
+    this.selectedValue = this.countSelectedTargets()
+    this.displayTarget.innerHTML = this.selectedValue
+  }
+
+  recount() {
+    this.selectedValue = this.countSelectedTargets()
+    this.displayTarget.innerHTML = this.selectedValue
+  }
+
+  countSelectedTargets() {
+    const selectedTargets = this.counterTargets.filter((el) => {
+      return el.firstElementChild.disabled
+    })
+    return this.counterTargets.length - selectedTargets.length
```

Comment:
Naming is a bit misleading ... `selectedTargets` appears to be all the _deselected_ hidden fields?  Maybe invert the above filter to `!disabled`, and then just return `.length`?